web/styles/settings.css

```diff
@@ -1698,20 +1708,15 @@ $option_title_width: 180px;
     display: inline-block;
 }
 
-.collapse-settings-btn {
-    margin: 10px 0 0 10px;
+/* .collapse-settings-btn,
```

Comment:
Looks like these lines are just commented out. I'm guessing they should be removed entirely.

Reply:
That was left by mistake, fixed it, thanks!